Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -151,10 +164,34 @@ private void UpdateRaycast()
             {
                 if (oldFocusedObject != null)
                 {
+                    IInteractable oldInteractable = oldFocusedObject.GetComponent<IInteractable>();
```

Comment:
Cache the IInteractable like I did in my test bed.  We might need to keep a queue of some sort to keep the cache from getting so large that the cache lookup is expensive, but for now a simple cache should be good enough.